parlai/mturk/core/agents.py

```diff
@@ -22,154 +24,6 @@
 # relative to heartbeats. This will come with more thorough testing.
 
 
-class AssignState:
```

Comment:
lol please explain wtf is going on

Reply:
It seems like this has to happen in order to get MTurk tests to not import torch =/ Honestly, I'm not sure if it's worth it

Reply:
There are two ways that MTurk tests import torch, and moving AssignState to shared_utils.py fixes one of them

Reply:
The other way can be fixed by moving the two `metrics` imports in `parlai.mturk.code.agents` into the specific methods that use them, because a lot of MTurk code relies on importing `Agent`. This is a lot of import/function shuffling, so lemme know if you still think it's worth it to avoid this torch import